integration_tests/pages/page.ts

```diff
@@ -20,7 +20,7 @@ export default abstract class Page {
   manageDetails = (): PageElement => cy.get('[data-qa=manageDetails]')
 
   clickSubmit(): void {
-    cy.get('button').click()
+    cy.get('a').contains('Continue').click()
```

Comment:
I think if you set `matchCase` to `false` as an option here it would pass the rest of the integration tests? https://docs.cypress.io/api/commands/contains#Arguments 

I seem to recall Ed making some changes about checking for exact matches somewhere but I can't find it 🤔

Reply:
huh, ok thanks :)